docs/docs/getting_started/Import.mdx

```diff
@@ -20,7 +20,7 @@ const dynamoose = require("dynamoose");
 <TabItem value="esmodules">
 
 ```js
-import { dynamoose } from "dynamoose";
+import dynamoose from "dynamoose";
```

Comment:
```suggestion
import * as dynamoose from "dynamoose";
```

Reply:
Otherwise this would throw an error in TypeScript: `This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.`

Reply:
ah yep, I do always set esModuleInterop. Good catch. Will update the PR a little bit later today.